include/SmartField.h

```diff
@@ -190,80 +190,85 @@ class SmartField<FieldType, tags::DEVICE, ACCESS>
   //************************************************************
   // Device functions
   //************************************************************
-  KOKKOS_INLINE_FUNCTION
+  KOKKOS_FORCEINLINE_FUNCTION
   unsigned get_ordinal() const { return stkField_.get_ordinal(); }
 
   // --- Default Accessors
   template <typename Mesh, typename A = ACCESS>
-  KOKKOS_INLINE_FUNCTION std::enable_if_t<!std::is_same<A, READ>::value, T>&
-  get(const Mesh& ngpMesh, stk::mesh::Entity entity, int component) const
+  KOKKOS_FORCEINLINE_FUNCTION
+    std::enable_if_t<!std::is_same<A, READ>::value, T>&
```

Comment:
```suggestion
    std::enable_if_t<!std::is_same_v<A, READ>, T>&
```
and all other occurrences of `std::is_same`.

Reply:
Updated.